backstop_data/engine_scripts/puppet/search/actions/export-contact-save-fields.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const Page = require('../../page-objects/crm-page.js');
+
+module.exports = async (engine, scenario, vp) => {
+  const page = await Page.build(engine, scenario, vp);
+
+  await require('./export-contact')(engine, scenario, vp);
+  await engine.click('#CIVICRM_QFID_2_exportOption');
+  await page.clickAndWaitForNavigation('#_qf_Select_next-bottom');
+
+  await page.clickSelect2Option('#s2id_autogen3', 'Addressee');
```

Comment:
This is not working in my local, as it takes some time to make `#s2id_autogen3` to be displayed. So please add the following line before clicking select 2.
`await engine.waitForSelector('#s2id_autogen3');`.

Reply:
Added: https://github.com/compucorp/backstopjs-config/pull/52/commits/cdac7ddc7ec44042ed66d3344db1f5559c3a5574